Makefile

```diff
@@ -354,7 +354,7 @@ lint-fix:
 	@if docker ps -a --format '{{.Names}}' | grep -Eq "^${containerMarkdownLintFix}$$"; then docker start -a $(containerMarkdownLintFix); else docker run --name $(containerMarkdownLintFix) -i -v "$(CURDIR):/work" $(markdownLintImage) . --fix; fi
 
 lint-go:
-	echo $(GIT_DIFF)
+	test -n "`golangci-lint version | awk '$4 >= 1.42'`"
```

Comment:
alternative would be to use `go run` in a "global" (not module) way, but it take much longer:
```
time go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1 version
3.44s user 0.58s system 198% cpu 2.032 total
```
Without warmup, it will take more than "5s". This is version - so just loading the right golangci-lint without doing any linting.